staging/src/k8s.io/cloud-provider/app/testing/testserver.go

```diff
@@ -84,13 +85,13 @@ func StartTestServer(t Logger, customFlags []string) (result TestServer, err err
 	if err != nil {
 		return TestServer{}, err
 	}
-	all, disabled := app.KnownControllers(), app.ControllersDisabledByDefault.List()
+	all := []string{"cloud-node", "cloud-node-lifecycle", "service", "route"}
```

Comment:
will anything notice if this drifts? does it matter if it does?

Reply:
It doesn't matter even if we pass an empty string array. Used for validate: `https://github.com/kubernetes/kubernetes/blob/bd95fb101dd83cfca9d95e55b2f0d29720016e51/staging/src/k8s.io/controller-manager/options/generic.go#L90`